dbms/tests/queries/0_stateless/00386_has_column_in_table.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS has_column_in_table_test;
```

Comment:
You should create tables in database `test`.
No drop table (cleanup) at end of test.